common/buildings/00_capital_buildings.txt

```diff
@@ -247,6 +275,12 @@ building_capital = {
 
 	resources = {
 		category = planet_buildings
+		produces = {
+			trigger = { exists = owner 
```

Comment:
I might be wanting too much, but can you add a linebreak after trigger = {

It is making it really hard to read.